apollo/actions/applyActionResponse.tsx

```diff
@@ -546,6 +546,7 @@ export default function applyActionResponse(
     }
     case 'BeginGame':
     case 'SecretDiscovered':
+    case 'OptionalWin':
```

Comment:
This will not actually apply the action properly! You need to move this case up to where `case 'GameEnd':` is to make sure it calls `applyGameOverActionResponse`.

Reply:
Ah I see. Just to make sure we're on the same page though, this is for the consumers of `applyActionResponse()`, right? Would it be possible for that consumer to call `applyGameOverActionResponse()` instead? Or maybe it's impossible to do so in some cases?